diff -ur 3dwm-0.3.1/config/macros/utils.m4 3dwm-0.3.1-/config/macros/utils.m4
--- 3dwm-0.3.1/config/macros/utils.m4	Mon May 28 14:47:24 2001
+++ 3dwm-0.3.1-/config/macros/utils.m4	Mon Feb  3 16:42:23 2003
@@ -889,5 +889,5 @@
 fi
 ])dnl
 
-divert
+divert(0)
 
diff -ur 3dwm-0.3.1/config/macros/mdl_have_opengl.m4 3dwm-0.3.1-/config/macros/mdl_have_opengl.m4
--- 3dwm-0.3.1/config/macros/mdl_have_opengl.m4	Mon Aug 20 15:48:53 2001
+++ 3dwm-0.3.1-/config/macros/mdl_have_opengl.m4	Mon Feb  3 16:53:35 2003
@@ -73,10 +73,10 @@
   AC_CACHE_CHECK([for OpenGL], mdl_cv_have_OpenGL,
   [
 dnl Check for Mesa first, unless we were asked not to.
-    AC_HELP_STRING([--with-Mesa],
-                   [Prefer the Mesa library over a vendors native OpenGL library (default=yes)],
-                   with_Mesa_help_string)
-    AC_ARG_ENABLE(Mesa, $with_Mesa_help_string, use_Mesa=$enableval, use_Mesa=yes)
+    AC_ARG_ENABLE(Mesa, 
+    	AC_HELP_STRING([--with-Mesa],
+                   [Prefer the Mesa library over a vendors native OpenGL library (default=yes)])
+    [use_Mesa=$enableval], [use_Mesa=yes])
 
     if test x"$use_Mesa" = xyes; then
        GL_search_list="MesaGL   GL"
diff -ur 3dwm-0.3.2/aclocal.m4 3dwm-0.3.2--/aclocal.m4
--- 3dwm-0.3.2/aclocal.m4	Mon Feb  3 21:30:17 2003
+++ 3dwm-0.3.2--/aclocal.m4	Mon Feb  3 21:29:33 2003
@@ -967,7 +967,7 @@
 				fi
 			fi
 			msg=enabled
-			AC_DEFINE(DEBUG)
+			AC_DEFINE(DEBUG, 1, [debugging])
 		else
 			msg=disabled
 		fi
@@ -1432,7 +1432,7 @@
 CFLAGS=`echo "$CFLAGS" | sed -e 's/-O3//g'`
 CFLAGS="$CFLAGS -O0 -pg"
 LDFLAGS="$LDFLAGS -pg"
-AC_DEFINE(profile)
+AC_DEFINE(profile, 1, [profiling])
 msg="enabled"
 ],[
 msg="disabled"
diff -ur 3dwm-0.3.2/config/macros/compiler.m4 3dwm-0.3.2--/config/macros/compiler.m4
--- 3dwm-0.3.2/config/macros/compiler.m4	Mon May 28 14:47:24 2001
+++ 3dwm-0.3.2--/config/macros/compiler.m4	Mon Feb  3 21:28:54 2003
@@ -99,7 +99,7 @@
 				fi
 			fi
 			msg=enabled
-			AC_DEFINE(DEBUG)
+			AC_DEFINE(DEBUG, 1, [debugging])
 		else
 			msg=disabled
 		fi
diff -ur 3dwm-0.3.2/config/macros/utils.m4 3dwm-0.3.2--/config/macros/utils.m4
--- 3dwm-0.3.2/config/macros/utils.m4	Mon Feb  3 21:30:17 2003
+++ 3dwm-0.3.2--/config/macros/utils.m4	Mon Feb  3 21:29:14 2003
@@ -381,7 +381,7 @@
 CFLAGS=`echo "$CFLAGS" | sed -e 's/-O3//g'`
 CFLAGS="$CFLAGS -O0 -pg"
 LDFLAGS="$LDFLAGS -pg"
-AC_DEFINE(profile)
+AC_DEFINE(profile, 1, [profiling])
 msg="enabled"
 ],[
 msg="disabled"
diff -ur 3dwm-0.3.2/configure.in 3dwm-0.3.2--/configure.in
--- 3dwm-0.3.2/configure.in	Mon Feb  3 21:30:17 2003
+++ 3dwm-0.3.2--/configure.in	Mon Feb  3 21:28:04 2003
@@ -37,9 +37,9 @@
 AM_INIT_AUTOMAKE($PACKAGE, $VERSION)
 
 dnl Export defines
-AC_DEFINE_UNQUOTED(MAJOR_VERSION)
-AC_DEFINE_UNQUOTED(MINOR_VERSION)
-AC_DEFINE_UNQUOTED(PATCH_LEVEL)
+AC_DEFINE_UNQUOTED(MAJOR_VERSION, $MAJOR_VERSION, [major version])
+AC_DEFINE_UNQUOTED(MINOR_VERSION, $MINOR_VERSION, [minor version])
+AC_DEFINE_UNQUOTED(PATCH_LEVEL, $PATCH_LEVEL, [patch level])
 
 dnl AC_HEADLINE(dnl
 dnl	3Dwm, The Three-Dimensional User Environment, dnl
